lighthouse-cli/run.ts

```diff
@@ -27,13 +29,36 @@ interface LighthouseError extends Error {
   code?: string
 }
 
+function parseChromeFlags(flags: string) {
+  let args = yargs(flags).argv;
+  const allKeys = Object.keys(args);
+
+  // Remove any "mangled" arguments (args that contain dashes are duplicated as
+  // camelCased variants, but the dashed version remains)
+  let keysToFilter = allKeys.filter(k => k.indexOf('-') !== -1)
```

Comment:
maybe to make this easier to follow we could just do a basic filter and map in one go?

```js
return Object.keys(args)
  .filter(key => {
    return key !== '_' &&
      !key.startsWith('$') &&
      !/[A-Z]/.test(key) // no camelcase keys (can also just use what you have now :) )
  })
  .map(key => {
    return args[key] === true ? ...
  })
```